packages/protocol/contracts/governance/ReleaseGoldInstance.sol

```diff
@@ -254,6 +256,16 @@ contract ReleaseGoldInstance is UsingRegistry, ReentrancyGuard, IReleaseGoldInst
     emit DistributionLimitSet(beneficiary, maxDistribution);
   }
 
+  /**
+   * @notice Sets the beneficiary of the instance
+   * @param newBeneficiary The address of the new beneficiary
+   */
+  function setBeneficiary(address payable newBeneficiary) external onlyBeneficiary {
```

Comment:
Do we have tests for this contract?

Reply:
Yes, does it not show in the diff for you?

Reply:
I see it, thanks!